### DISCUSSION TITLE
No valid user created or default window layout found. Please revert factory settings to restore the default layouts. Unity-Supabase

### DISCUSSION THREAD

**Discussion by oztrodev** [supabase-community/supabase-csharp#122] [Q&A] [answered]
I Clone https://github.com/wiverson/supabase-unity-template and I have installed the unity version 2022.3.9f1, but when I try to open, appear this Message:

![image](https://github.com/supabase-community/supabase-csharp/assets/2772025/9165342f-05b2-4da0-a5e7-b0dc99d34ddb)

How can I open this proyect? thanks

**Comment by wiverson** [comment, marked as answer]
Just hit revert factory settings and give it a try.
